chapters/qa-03.mdx

```diff
@@ -57,28 +61,28 @@ Poměr slučovací: „Podejte nám zprávu písemně nebo telefonicky”
 | **Podejte nám zprávu písemně** | **Podejte nám zprávu telefonicky** | **Podejte nám zprávu písemně nebo telefonicky** |                                                                            |
 | :----------------------------: | :--------------------------------: | :---------------------------------------------: | :------------------------------------------------------------------------: |
 |        1 (true, pravda)        |          1 (true, pravda)          |                1 (true, pravda)                 |                      Zprávu můžu podat oběma způsoby                       |
-|        1 (true, pravda)        |         0 (false,nepravda)         |                1 (true, pravda)                 |                       Zprávu můžu podat jen písemně                        |
-|       0 (false,nepravda)       |          1 (true, pravda)          |                1 (true, pravda)                 |                     Zprávu můžu podat jen telefonicky.                     |
-|       0 (false,nepravda)       |         0 (false,nepravda)         |               0 (false,nepravda)                | Pokud podáš zprávu jinak než písemně nebo telefonicky, tak nebude přijata. |
+|        1 (true, pravda)        |         0 (false, nepravda)         |                1 (true, pravda)                 |                       Zprávu můžu podat jen písemně                        |
```

Comment:
Do you have Prettier install? Formatting is now broken (at least in Github).

Reply:
Preview on local is without change
![image](https://user-images.githubusercontent.com/106968514/216438151-e21b77e9-89b3-4bfd-9d69-14162437d9e2.png)